Assets/Scripts/Manager/PlayerManager.cs

```diff
@@ -1,12 +1,24 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Events;
 
 public class PlayerManager : MonoBehaviour
 {
     public static PlayerManager Instance { get; private set; }
+    public static event UnityAction<GameObject> OnPlayerChanged;
+
+    public GameObject playerGameObject
```

Comment:
I prefer naming it to just `Player` and `_player` for reduced redundancy.

`public GameObject Player`